contracts/utils/MultiRateLimited.sol

```diff
@@ -32,6 +32,13 @@ contract MultiRateLimited is RateLimited, IMultiRateLimited {
     /// @notice max buffer cap allowable by non governor
     uint256 public maxBufferCap;
 
+    /// @param coreAddress address of the core contract
+    /// @param _maxRateLimitPerSecond maximum amount of fei that can replenish per second ever, this amount cannot be changed by governance
+    /// @param _rateLimitPerSecond maximum rate limit per second per address
+    /// @param _maxRateLimitPerSecondMRL maximum rate limit per second per address in multi rate limited
```

Comment:
This var name doesn't make much sense to me; instead of "the maximum in [some sub-contract that we don't care about]", the name should reflect the functionality of the var. Something like "_maxRateLimitPerSecondPerAddress" would clear this up quickly